packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -164,3 +167,40 @@ class _AlternateCheckedModeBanner extends StatelessWidget {
     );
   }
 }
+
+class _BulletSpacer extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: const BoxConstraints.tightFor(width: 24.0, height: 48.0),
+      alignment: Alignment.center,
+      child: const Text('•'),
+    );
+  }
+}
+
+class ReportBugAction extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: () async {
+        // TODO(devoncarew): Support analytics.
+        // ga.select(ga.devToolsMain, ga.feedback);
+
+        const reportIssuesUrl = 'https://github.com/flutter/devtools/issues';
+
+        if (await url_launcher.canLaunch(reportIssuesUrl)) {
+          await url_launcher.launch(reportIssuesUrl);
+        }
+      },
+      child: Container(
+        constraints: const BoxConstraints.tightFor(width: 48.0, height: 48.0),
```

Comment:
just setting width and height on the container should have the same effect and be a bit clearer.

Reply:
updated